src/Smd/Service.php

```diff
@@ -166,10 +182,8 @@ public function setName($name)
 
     /**
      * Retrieve name.
-     *
-     * @return string
      */
-    public function getName()
+    public function getName(): string
```

Comment:
Careful: adding return types to open classes is a BC break - please avoid doing this on non-`final` classes

Reply:
Agreed, reverted.